.gitignore

```diff
@@ -2,5 +2,10 @@
 /node_modules
 
 # security related
-db/password.txt
-.env
\ No newline at end of file
+.env
+
+# IDE related
+.DS_Store
+
+# personal files
+screenshots/
```

Comment:
maybe a /data folder with all others folder as subfolders might be a good idea (that can be then later a mount point?)